src/main/java/fi/aalto/cs/apluscourses/intellij/model/Scala3Sdk.java

```diff
@@ -29,6 +29,9 @@ public Scala3Sdk(@NotNull String name,
   @Override
   @NotNull
   protected String @NotNull [] getClassRoots() {
-    return new String[] {"scala3-library_3-" + scalaVersion + ".jar"};
+    return new String[] {
+        "scala-library-2.13.8.jar",
```

Comment:
You can call `getJarFiles()` and find the filename starting with "scala-library" to avoid hardcoding the version number. 

Perhaps skip Scala2 dependency altogether, if no such JAR is found (as I imagine they could get rid of that in the future).

(Consider adding `streamJarFiles()` to IntelliJLibrary to make handling them smoother.)